Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -415,6 +415,7 @@ void DrawMol::extractMolNotes() {
         didIt = true;
         break;
       }
+      delete annot;
```

Comment:
It think it might be preferable to do:
```
      std::unique_ptr<DrawAnnotation> annot(
          new DrawAnnotation(note, TextAlignType::START, "note", 1.0, locs[i],
                             drawOptions_.annotationColour, textDrawer_));
      // Put it into the legends_, because it's already in draw coords, so
      // shouldn't be treated by changeToDrawCoords.
      if (!doesNoteClash(*annot)) {
        legends_.emplace_back(std::move(annot));
        didIt = true;
        break;
      }
    }
    if (!didIt) {
      // There was nowhere to put it that didn't clash, so live with it.
      std::unique_ptr<DrawAnnotation> annot(
          new DrawAnnotation(note, TextAlignType::START, "note", 1.0, locs[0],
                             drawOptions_.annotationColour, textDrawer_));
      legends_.emplace_back(std::move(annot));
    }
```